Corgibytes.Freshli.Cli/Functionality/Git/GitSource.cs

```diff
@@ -149,14 +149,47 @@ private void Checkout(string gitPath)
 
     private void Pull(string gitPath)
     {
+        var stdOutBuffer = new StringBuilder();
+        var branch = Branch;
+        if (Branch == null)
+        {
+            branch = FetchCurrentBranch(gitPath);
+        }
+
         var stdErrBuffer = new StringBuilder();
         var command = CliWrap.Cli.Wrap(gitPath).WithArguments(
                 args => args
                     .Add("pull")
                     .Add("origin")
-                    .Add(Branch ?? "")
+                    .Add(branch ?? "")
             )
             .WithWorkingDirectory(Directory.FullName)
+            .WithStandardOutputPipe(PipeTarget.ToStringBuilder(stdOutBuffer))
+            .WithStandardErrorPipe(PipeTarget.ToStringBuilder(stdErrBuffer));
+
+        using var task = command.ExecuteAsync().Task;
+        task.Wait();
+
+        var commandOutput = stdOutBuffer.ToString().Replace("\n", "");
```

Comment:
Wouldn't this mash output together? `First Line\nSecond Line` would become `First LineSecond Line`, which isn't terribly readable. At minimum, we should replace `\n` with a space.